src/main/java/data/Text/Main.java

```diff
@@ -0,0 +1,30 @@
+package data.Text;
+
+import javax.imageio.IIOException;
+import java.io.File;
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        try {
+            File file = new File("data.txt");
+            if (!file.exists()) {
+                file.createNewFile();
+            }
+        } catch (IIOException exception) {
+            exception.printStackTrace();
+        }
+
+        String pathToFile = "C:\\Users\\Ali\\IdeaProjects\\Github\\target\\data.txt";
```

Comment:
w tym momencie u mnie to nie dziala. Przeczytaj prosze:
https://docs.oracle.com/javase/tutorial/essential/io/path.html